app/src/main/java/com/oracle/javafx/scenebuilder/app/SceneBuilderApp.java

```diff
@@ -1011,7 +1013,7 @@ private void showVersionNumberFormatError(DocumentWindowController dwc) {
         // in development so we don't localize the strings
         alert.setTitle("Error");
         alert.setHeaderText(I18N.getString("check_for_updates.alert.headertext"));
-        alert.setContentText("Version number format not supported. Maybe using SNAPSHOT or RC versions.");
+        alert.setContentText("Update check is disabled in development environment.");
```

Comment:
The new snapshot check doesn't throw NFE. If it is not a snapshot, and while checking the version there is a NFE, the error message should still be `Version number format not supported`.

Reply:
The new snapshot will not throw NFE during startup. However, it will throw if a user goes selects "Check for updates.." from the Help menu.